src/parse/rules/sh_rules.build_defs

```diff
@@ -110,6 +111,7 @@ def sh_test(name:str, src:str=None, labels:list&features&tags=None, data:list=No
         srcs=[src or test],
         data=data,
         deps=deps,
+        worker=worker,
```

Comment:
I don't think you can do this? `build_rule` doesn't take `worker` directly?

Reply:
More oops >.<